Cargo.toml

```diff
@@ -1,6 +1,32 @@
+[package]
+name = "tower"
+version = "0.1.0"
+authors = ["Lucio Franco <luciofranco14@gmail.com>"]
```

Comment:
This is super nitpicky, but I'd probably expand this list.
Or set it to the relatively generic `Carl Lerche <me@carllerche.com>` :joy: